cyber-threat-intelligence/examples/07-external-reference.tql

```diff
@@ -0,0 +1,25 @@
+# Example 07:
+
+# From the Example 06, we can see that some attack patterns are used by intrusion sets.
```

Comment:
```suggestion
# From Example 06, we can see that some attack patterns are used by intrusion sets.
```

Reply:
Can we see that? I didn't see any intrusion sets in either Example or Exercise 06.